python/nano/notebooks/hpo/custom.ipynb

```diff
@@ -0,0 +1,1609 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**In this guide we will demonstrate how to use Nano AutoML to auto tune an machine learning model and handle the whole model development process easily.**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction\n",
+    "Nano provides hpo as an solution for fast and high-performance hyperparameter tuning. Users can import hano hpo following the standard tensorflow API without changing their own code. By setting the search space and running model.search(), the model will contain the best hyperparameters.\n",
+    "\n",
+    "Nano hpo supports sequential, fuctional and custom keras API.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step0: Prepare Environment\n",
+    "We recommend using conda to prepare the environment.\n",
+    "conda create -n my_env python=3.7\n",
+    "\n",
+    "conda activate my_env\n",
+    "\n",
+    "pip install --pre --upgrade bigdl"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step1: Init Nano AutoML\n",
+    "Let's start out by importing Nano AutoML and enable its tensorflow API globally by automl.hpo_config.enable_hpo_tf().\n",
```

Comment:
break this large cell into two steps: one for "Init Nano AutoML" and one for "Prepare Data"